KalturaMediaServer/src/com/kaltura/media/server/managers/KalturaCuePointsManager.java

```diff
@@ -118,8 +120,16 @@ public void load(final List<String> entryIds, boolean periodic) {
 						}
 						
 						// create sync-point 30 seconds before the trigger time
-						date.setTime((cuePoint.triggeredAt / 1000) - 30000);
+						long scheduledTime = (cuePoint.triggeredAt / 1000) - 30000;
```

Comment:
If you set value that is smaller than 1980 using the API, the server will translate it to relative time and will add the specified number to time(), that's why this number can't be small.
If you know of a specific case that it was small, then it's a bug, and we should solve it, offline.